misc/formatter/modules/formatter-core/src/test/java/org/ballerinalang/formatter/core/configurations/LocalConfigurationTest.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023, WSO2 LLC. (http://wso2.com) All Rights Reserved.
```

Comment:
There are around 20 such cases. Can we update them?

```suggestion
 * Copyright (c) 2024, WSO2 LLC. (http://wso2.com).
```

Reply:
I think that year is the commited year of the code.

Reply:
Updated